clients/privacy-center/packages/fides-consent/src/consent-config.json

```diff
@@ -0,0 +1,31 @@
+{
+  "options": [
```

Comment:
This is auto-generated, isn't it? Did a .gitignore get missed somewhere maybe?

Reply:
We actually need to remove the entire `clients/privacy-center/packages/` directory, as that's basically empty now that we've extracted out `fides-consent` from privacy center